external/storm-jdbc/src/main/java/org/apache/storm/jdbc/common/Column.java

```diff
@@ -27,18 +26,16 @@
  *  ----------------------------
  * </pre>
  *
- * The following class can be used to represent the data in the table as
+ * <p>The following class can be used to represent the data in the table as
  * <pre>
- * List<List<Column>> rows = new ArrayList<List<Column>>();
- * List<Column> row1 = Lists.newArrayList(new Column("UserId", 1, Types.INTEGER), new Column("UserName", "Foo", Types.VARCHAR))
- * List<Column> row1 = Lists.newArrayList(new Column("UserId", 2, Types.INTEGER), new Column("UserName", "Bar", Types.VARCHAR))
+ * List&lt;List&lt;Column&gt;&gt; rows = new ArrayList&lt;List&lt;Column&gt;&gt;();
```

Comment:
I think we should use `@{code}` to escape the special characters here https://reflectoring.io/howto-format-code-snippets-in-javadoc/